libraries/aws_iam_root_user.rb

```diff
@@ -29,4 +39,8 @@ def to_s
   def summary_account
     @summary_account ||= @client.get_account_summary.summary_map
   end
+
+  def virtual_mfa_devices
```

Comment:
We should also update the docs file.

Reply:
We should also update the integration test in the "default" test suite.

Reply:
This PR doesn't actually add a matcher for a hardware MFA, unless I'm missing that?

Reply:
The benchmark for this issue is to determine if hardware MFA is enabled for the root account, which is done by checking `aws iam list-virtual-mfa-devices`, and if there's an MFA with the serial number ` "arn:aws:iam::<aws_account_number>:mfa/root-account-mfa-device"` then the MFA is virtual, not hardware, so it's not compliant.
The issue #73 maybe should have said `Add has_a_hardware_mfa_device? to root_account` instead of `Add has_virtual_mfa_devices? to root_account`, but one is just the inverse of the other.  Originally I had this set up as `has_hardware_mfa_enabled` instead, but changed it after to match the issue title, I can revert the last 2 commits and update the name from `has_hardware_mfa_enabled` to  `has_a_hardware_mfa_device`. Sound good @clintoncwolfe ?

I can update the doc as well. As for the integration tests, we couldn't find a way to do this with terraform, adding a hardware MFA device.